src/status_im2/contexts/activity_center/notification/contact_requests/view.cljs

```diff
@@ -0,0 +1,113 @@
+(ns status-im2.contexts.activity-center.notification.contact-requests.view
+  (:require [quo2.core :as quo]
+            [react-native.core :as rn]
+            [status-im2.constants :as constants]
+            [status-im2.contexts.activity-center.notification.common.view :as common]
+            [status-im2.contexts.activity-center.notification.contact-requests.events]
+            [utils.datetime :as datetime]
+            [utils.i18n :as i18n]
+            [utils.re-frame :as rf]))
+
+(defn outgoing-contact-request-view
+  [{:keys [id chat-id message last-message] :as notification}]
+  (let [{:keys [contact-request-state] :as message} (or message last-message)]
+    (if (= contact-request-state constants/contact-request-message-state-accepted)
+      [quo/activity-log
+       {:title     (i18n/label :t/contact-request-was-accepted)
+        :icon      :main-icons2/add-user
```

Comment:
The qualification for icons should be `:i`. `:main-icons2` is old stuff. There are other instances of this problem in this file.